src/studio/src/designer/backend/gulpfile.js

```diff
@@ -29,82 +42,85 @@ const cssDashboardFile = '../frontend/dist/dashboard/dashboard.css';
+    src: 'node_modules/json-editor/dist/*.js',
+    dest: 'wwwroot/designer/js/lib',
   },
   {
-    src: "node_modules/select2/dist/js/select2.full.js",
-    dest: "wwwroot/designer/js/lib"
+    src: 'node_modules/select2/dist/js/select2.full.js',
+    dest: 'wwwroot/designer/js/lib',
   },
   {
-    src: "node_modules/select2/dist/css/select2.css",
-    dest: "wwwroot/designer/css/lib"
+    src: 'node_modules/select2/dist/css/select2.css',
+    dest: 'wwwroot/designer/css/lib',
   },
   {
-    src: "node_modules/jquery/dist/*.js",
-    dest: "wwwroot/designer/js/lib"
+    src: 'node_modules/jquery/dist/*.js',
+    dest: 'wwwroot/designer/js/lib',
   },
   {
-    src: "node_modules/requirejs/require.js",
-    dest: "wwwroot/designer/js/lib/"
+    src: 'node_modules/requirejs/require.js',
+    dest: 'wwwroot/designer/js/lib/',
   },
   {
-    src: "node_modules/underscore/*.js",
-    dest: "wwwroot/designer/js/lib"
+    src: 'node_modules/underscore/*.js',
+    dest: 'wwwroot/designer/js/lib',
   },
   {
-    src: "node_modules/requirejs-text/*.js",
-    dest: "wwwroot/designer/js/lib"
+    src: 'node_modules/requirejs-text/*.js',
+    dest: 'wwwroot/designer/js/lib',
   },
   {
-    src: "node_modules/js-beautify/js/lib/beautify*.js",
-    dest: "wwwroot/designer/js/lib/"
+    src: 'node_modules/js-beautify/js/lib/beautify*.js',
+    dest: 'wwwroot/designer/js/lib/',
   },
   {
-    src: "node_modules/sightglass/*.js",
-    dest: "wwwroot/designer/js/lib/"
+    src: 'node_modules/sightglass/*.js',
+    dest: 'wwwroot/designer/js/lib/',
   },
   {
-    src: "node_modules/rivets/dist/*.js",
-    dest: "wwwroot/designer/js/lib/"
+    src: 'node_modules/rivets/dist/*.js',
+    dest: 'wwwroot/designer/js/lib/',
   },
   {
-    src: "node_modules/jquery-validation-unobtrusive/dist/*.js",
-    dest: "wwwroot/designer/js/lib"
+    src: 'node_modules/jquery-validation-unobtrusive/dist/*.js',
+    dest: 'wwwroot/designer/js/lib',
   },
   {
-    src: "node_modules/jquery-validation/dist/*.js",
-    dest: "wwwroot/designer/js/lib"
+    src: 'node_modules/jquery-validation/dist/*.js',
+    dest: 'wwwroot/designer/js/lib',
   },
   {
-    src: "node_modules/popper.js/dist/umd/*.*.js",
-    dest: "wwwroot/designer/js/lib"
+    src: 'node_modules/popper.js/dist/umd/*.*.js',
+    dest: 'wwwroot/designer/js/lib',
   },
   {
-    src: "node_modules/monaco-editor/min/**/*.*",
-    dest: "wwwroot/designer/js/lib/monaco-editor"
+    src: 'node_modules/monaco-editor/min/**/*.*',
+    dest: 'wwwroot/designer/js/lib/monaco-editor',
   },
   {
-    src: "node_modules/bootstrap-list-filter/bootstrap-list-filter.min.js",
-    dest: "wwwroot/designer/js/lib"
-  }
+    src: 'node_modules/bootstrap-list-filter/bootstrap-list-filter.min.js',
+    dest: 'wwwroot/designer/js/lib',
+  },
 ];
 
 function copyNodeModulePackages(cb) {
-  copyGlobs.map(copyGlob => gulp.src(copyGlob.src).pipe(gulp.dest(copyGlob.dest)));
+  copyGlobs.map((copyGlob) =>
```

Comment:
From Sonar, this could be a forEach,
but there are several other things we could do to clean up this file, like most of the `dest` above are the same, so don't bother :)

Reply:
Yeah, I noticed as well. Ill change this before merging :) 

I think long term it would be good to get rid of this copying of files around as much as possible. Its confusing, and easy to get wrong (ref the incident with incorrect path in Dockerfile). This would probably mean some heavy lifting on architecture, so not sure when we will get time though.